tests/test_model.py

```diff
@@ -0,0 +1,30 @@
+"""Test for merging MappingSetDataFrames."""
+
+import unittest
+
+from sssom_schema import Mapping
+
+
+class TestModel(unittest.TestCase):
+    """A test case for making sure the model works as intended."""
+
+    def test_mapping_throws_value_error(self):
+        """Test if instantiating Mapping() fails when required elements are missing."""
+        mdict_missing = dict(
+            subject_id="ID:123"
+        )  # This is missing object_id, predicate_id, mapping_justification
+        mdict_complete = dict(
+            subject_id="AID:123",
+            object_id="BID:123",
+            predicate_id="skos:exactMatch",
+            mapping_justification="semapv:LexicalMatching",
+        )
+
+        with self.assertRaises(ValueError):
```

Comment:
can we check that the value error is giving back feedback about what's missing? Also, shouldn't this be a type error? These are probably issues with linkml upstream, so let's just make notes about them

Reply:
I added a check to that end, but it is not perfect - the LinkML error only returns the first problem, not _all_ problems, in the error message..